### PR TITLE
QMAPS-516 locale hours on POI panel

### PR DESCRIPTION
**Pull request by patfrat**
Translate also opening hours in local format

### CODE REVIEW

File: src/adapters/osm_schedule.js [hunk 1]
```diff
@@ -31,10 +31,27 @@ function nextTransitionTime(seconds, nextTransitionDate) {
   return false
 }
 
+function hourToDate(hour) {
+  return new Date(new Date().toDateString() + ' ' + hour)
+}
+
+function translateOpeningHours(hours) {
```
Comment (Julien-laville) [inline src/adapters/osm_schedule.js]: (Law of triviality) it's more a localisation than a translation

File: src/adapters/osm_schedule.js [hunk 2]
```diff
@@ -31,10 +31,27 @@ function nextTransitionTime(seconds, nextTransitionDate) {
   return false
 }
 
+function hourToDate(hour) {
+  return new Date(new Date().toDateString() + ' ' + hour)
```
Comment (Julien-laville) [inline src/adapters/osm_schedule.js]: I prefer template string

Comment (Julien-laville) [inline src/adapters/osm_schedule.js]: i dont trust date dateString parsing, can we use arithmetic instead ?
`return new Date(Date.now() + hour * 60)`

Comment (patfrat) [reply src/adapters/osm_schedule.js]: hour is a String : Date.now() + "06:30" * 60 > NaN

File: src/adapters/osm_schedule.js (hunk 1 above)
Comment (patfrat) [reply src/adapters/osm_schedule.js]: renamed toLocaleOpeningHours

File: src/adapters/osm_schedule.js (hunk 2 above)
Comment (patfrat) [reply src/adapters/osm_schedule.js]: done

File: src/adapters/osm_schedule.js [hunk 3]
```diff
@@ -31,10 +31,27 @@ function nextTransitionTime(seconds, nextTransitionDate) {
...
+  if(hours) {
+    return hours.map((hour) => {
+      let beginningHour = hourToDate(hour.beginning)
+      let endHour = hourToDate(hour.end)
+      return {
+        beginning: strftime(i18nDate.timeFormat, beginningHour),
+        end: strftime(i18nDate.timeFormat, endHour)
+      }
+    })
+  }
```
[the remark this answers is not in the document]
Comment (patfrat) [reply src/adapters/osm_schedule.js]: it displays "closed" but i have added a return with an empty array

**Pull request merged**